app/client/src/features/events/Moderation/ModeratorEventSettings.tsx

```diff
@@ -201,7 +201,7 @@ export const ModeratorEventSettings = ({ fragmentRef, className }: EventSettings
                             <ListItemAvatar>
                                 <Avatar src={avatar || undefined} />
                             </ListItemAvatar>
-                            <ListItemText primary={`${firstName} ${lastName}`} secondary={email} />
+                            <ListItemText primary={firstName ? `${firstName} ${lastName}` : 'Pending Registration...'} secondary={email} />
```

Comment:
For thoroughness, this should probably be generated with a `useMemo`.  There's a few cases here: If `firstName` is defined but `lastName` is not, we don't want to display "David Undefined".  Displaying "David" alone is probably okay.  So we should check whether or not lastName exists too.  Could do something like `const displayName = useMemo(() => ...);`